### ISSUE TITLE
Add a language selection screen on first startup

### ISSUE THREAD

**Issue by lotharsm** [libretro/RetroArch#4629]
Currently, RetroArch runs with English language selected on first startup. To change the language to the user's native language, the user has to use the menu. This might be challenging for some people that have (very) little to none knowledge of the English language.

Therefore I suggest adding a simple language selection menu on the first launch by default. After selecting the desired language, RetroArch should automatically restart with the selected language and keep this language as default for future launches.

In my opinion, this will improve usability for some users.

**Comment by fr500** [comment]
Yeah we should have a full blown first run wizard on the first run

**Comment by rootfather** [comment]
Yeah, this would be really great. Hopefully some skilled person can implement this.

**Comment by rootfather** [comment]
Any news on this? I noticed that we now have a Windows setup program. Could this be used to implement a simple language selection drop down menu during the installation that prepares a retroarch config file with the proper language settings?

**Comment by rootfather** [comment]
I addes a 25$ bounty on this issue. Hope this will attract your attention.

**Comment by gouchi** [comment]
@lotharsm RA is now selecting the user language automatically at first startup.

Do you think, we may close this issue ?

Thank you.

**Comment by zoltanvb** [comment]
I agree that it can be closed, language autodetection is in place since a while on most platforms and seems to bring no issue reports.
